addons/mail/static/src/core/common/message_model.js

```diff
@@ -252,7 +252,7 @@ export class Message extends Record {
     isPending = false;
 
     get hasActions() {
-        return !this.is_transient;
+        return !this.is_transient && !this.isPending;
```

Comment:
This reintroduce the problem that was fixed by https://github.com/odoo/odoo/pull/204958
The invisible area for quick action of pending messages requires `hasActions` currently in `message.xml` template, see `emptyQuickAction`.

Should either not adapt `hasActions` or should adapt template to handle `emptyQuickAction` even when `hasActions: false`. I would prefer the 1st approach, because this feels more natural and don't waste space for message without actions. Also pending messages do not have actions but probable future would have, e.g. quick action to cancel them, or to retry send them, or even optimistic behaviour to pre-trigger actions when it becomes genuine message.

Reply:
Ok, I think it's not a big deal to keep `hasActions` even if there is no action?

The main goal of the PR is to change the one in `canToggleStar` (and few others that made no sense while checking for other potential occurrences of the bug).

Reply:
In regards to the other bug, is it ok to not have the action button itself as long as `hasActions` is `true`?
Or do we need to play with visibility to keep the place used in the DOM?